ui/package.json

```diff
@@ -56,5 +56,8 @@
     "devDependencies": {
         "chalk-cli": "^5.0.0",
         "lerna": "^3.22.1"
+    },
+    "dependencies": {
+        "cross-spawn": "7.0.5"
```

Comment:
@sachaudh as mentioned on private channel, is it possible to delete this and still keep change below?